src/ulvdivide.jl

```diff
@@ -121,14 +121,15 @@ function _utransforms_kernel!(hssA::HssMatrix, Q::BinaryNode)
 end
 function _ltransforms_kernel!(hssA::HssMatrix, Q::BinaryNode)
   nk = size(Q.data,1)
-  hssA.D[1:nk, :] = trsm('L', 'L', 'N', 'N', 1., Q.data, hssA.D[1:nk,:])
-  hssA.U[1:nk, :] = trsm('L', 'L', 'N', 'N', 1., Q.data, hssA.U[1:nk,:])
+  hssA.D[1:nk, :] = trsm('L', 'L', 'N', 'N', eltype(hssA)(1), Q.data, hssA.D[1:nk,:])
```

Comment:
I'm not sure why you changed this to `eltype(hssA)(1)`? What is the reasoning behind this change?

Reply:
We have to ensure that the types are consistent in this function call, otherwise it throws an error: 
"no method matching trsm(::Char, ::Char, ::Char, ::Char, ::Float64, ::Matrix{Float32}, ::Matrix{Float32})"